Makefile

```diff
@@ -34,3 +37,23 @@ gh-pages:
 reindent:
 	git ls-files '*.ml' '*.mli' | xargs ocp-indent --syntax cstruct -i
 
+regenerate-files:
+	opam install --deps-only xapi -y
```

Comment:
I think, strictly speaking, we also need a depext here:
```
opam depext xapi -y
```

Reply:
True, but we are also asking for much more than what we really need. And soon will not be necessary anymore